README.md

```diff
@@ -10,21 +10,31 @@ UI assets and code for Lighthouse.
 
 ## Development
 
-Then, install our node dependencies, do a dev build, and optionally start the js watcher (which will automatically do a dev build when files change).
+### Setup
 
 * `npm install -g bower`
 * `bower install`
 * `npm install -g gulp`
 * `npm install`
-* `gulp dev build`
-* `gulp watch` (optional)
+
+### Doing things
+
+* `gulp build` default build (non-minified source) to `../lighthouse/static`
+* `gulp watch` optional, rebuilds on source changes
 * `gulp clean` removes built assets from the server static dir (see Gulpfile.js)
-* `gulp prod build` will do the same as `dev`, but with minified js
-* fire up a webserver from `lighthouse/static` and navigate to `localhost` at the port specified by the server
 
-New angular modules should follow the same pattern as `instances`. Generally speaking, each major piece of functionality should be pulled out into it's own module and given a unique name to link into the root module. (see `app/js/app.js`)
+Options:
+
+* `--prod` build minfied source to `../lighthouse/static`
+* `--gopath` build to `$GOPATH/src/github.com/lighthouse/lighthouse/static/`
```

Comment:
@theMagicalKarp These are the build updates I mentioned last week. If you're just hacking on the backend and want to build to your go path, `gulp build --gopath` is all you need. And, optionally, `gulp build --gopath --prod`